helpers/format.js

```diff
@@ -21,3 +21,10 @@ Handlebars.registerHelper('verbifyDomain', function(opts) {
   // console.log('verbifyDomain', domain)
   return domain && domain.replace('.to', ' to')
 })
+
+Handlebars.registerHelper('prettyCredit', function(credit) {
+  if (credit)
```

Comment:
@dbdr actually, would you mind changing all your conditionals to use curly braces?